pkg/controller/util.go

```diff
@@ -0,0 +1,351 @@
+package controller
+
+import (
+	core_util "github.com/appscode/kutil/core/v1"
+	meta_util "github.com/appscode/kutil/meta"
+	api "github.com/kubedb/apimachinery/apis/kubedb/v1alpha1"
+	cs "github.com/kubedb/apimachinery/client/clientset/versioned"
+	"github.com/kubedb/apimachinery/client/clientset/versioned/typed/kubedb/v1alpha1/util"
+	coreV1 "k8s.io/api/core/v1"
+	kerr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Remove Owner Reference from Snapshots
+func RemoveOwnerReferenceFromSnapshots(
```

Comment:
Move to https://github.com/kubedb/apimachinery/blob/master/client/clientset/versioned/typed/kubedb/v1alpha1/util/snapshot.go